test/rest.browser.test.js

```diff
@@ -29,7 +29,10 @@ describe('strong-remoting-rest', function() {
 
   // setup
   beforeEach(function() {
-    objects = RemoteObjects.create();
+    objects = RemoteObjects.create({
+      handleErrors: false,
```

Comment:
hmm, why `handleError` is set to false?

Reply:
Good catch. I was experimenting with different error-handler settings to get a more helpful test failure message. I think this line is not needed, I'll remove it.